.github/workflows/reviewdog.yml

```
@@ -0,0 +1,28 @@
+name: reviewdog
+
+on:
+  - pull_request
+
+jobs:
+  prettier:
+    name: prettier
```

I think we should try prevent "failing CI on forks" shenanigans:

```suggestion
    # do not run on forks
    if: github.repository == 'mdn/content'
    name: prettier
```

https://github.com/mdn/content/blob/9c1ffea194f0b8fa91effad569c9e7c439802b9d/.github/workflows/pr-labeler.yml#L7-L8